Document required workaround for LFS-enabled repositories (#44)
saadmk11/github-actions-version-updater: README.md

To know more about how to pass a secret to GitHub actions you can [Read GitHub Docs](https://docs.github.com/en/actions/reference/encrypted-secrets)

> ### A note about GitLFS

Reviewer: ```suggestion
### A note about Git Large File Storage (LFS)
```

> If your repository uses Git LFS, you will need to manually remove the LFS-related hook files, otherwise the action

Reviewer: ```suggestion
If your repository uses [Git LFS](https://git-lfs.github.com/), you will need to manually remove the LFS-related hook files, otherwise the action
```

will fail because Git will not be able to create a branch because the lfs executable is not installed inside the
container used by this action.

To work around this, just remove the hook files manually as an extra step **before** this action executes:

```yaml
# ...
jobs:
  build:
    runs-on: ubuntu-latest

    steps:
      - uses: actions/checkout@v2
        with:
          token: ${{ secrets.WORKFLOW_SECRET }}
          lfs: false

      - name: Remove LFS hooks

>         # This repository uses GitLFS, but it not being

Reviewer: ```suggestion
        # This repository uses Git LFS, but it not being
```
